### PR TITLE
chore: introduce pipeline for testing podman installation and podman …

### LINKED ISSUE #16: Integrate Podman Desktop Playwright e2e tests into workflow testing Podman Installation using GOAX on windows

### PR DESCRIPTION
**Pull request by odockal**
…desktop e2e tests. Fixes #16.

### CODE REVIEW

File: .github/workflows/podman-desktop-e2e-windows.yaml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+          quay.io/odockal/pde2e-builder:v0.0.1-snapshot  \
+              pd-e2e/run.ps1 \
+                  -targetFolder pd-e2e \
+                  -resultsFolder results \
+                  -fork ${{ inputs.fork }} \
+                  -branch ${{ inputs.branch }}
+        # check logs
+        podman logs -f pde2e-builder-run
+        
+    - name: Run podman desktop e2e
```
Comment (adrianriobo) [inline .github/workflows/podman-desktop-e2e-windows.yaml:91]: Can you add some comment explaining why we keep using this step here (maybe link to the issue on podman-desktop??) so we can ensure if it is solved we should remove the step from here

File: .github/workflows/podman-desktop-e2e-windows.yaml [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+          -v $PWD:/data:z \
+          quay.io/rhqp/podman-desktop-e2e:v1.1.0-windows-amd64  \
+              pd-e2e/run.ps1 \
+                  -targetFolder pd-e2e \
+                  -pdPath $pdPath \
+                  -junitResultsFilename pd-e2e-results.xml 
+        # Check logs 
+        podman logs -f pd-e2e-windows
+
+    - name: Run PDE2E Runner image
```
Comment (adrianriobo) [inline .github/workflows/podman-desktop-e2e-windows.yaml:117]: I guess this should be implemented here https://github.com/odockal/pde2e-runner but I see no code there, can you pushed to see what is doing?

File: .github/workflows/podman-desktop-e2e-windows.yaml [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+          -e RDP_PASSWORD=$(cat userpassword) \
+          quay.io/rhqp/frdp:v0.0.1
+        # Wait until the x session has been created
+        podman wait --condition running x-session
+        # Check logs for the x session
+        podman logs x-session
+
+    - name: Run PDE2E builder and setup image
+      run: |
+        podman run -d --name pde2e-builder-run \
```
Comment (adrianriobo) [inline .github/workflows/podman-desktop-e2e-windows.yaml:73]: From the script I see you build PD based on fork and branch so you would be able to build from PR, but should not be easier to just pass the PR-Number?

Also I see you install dependencies without choco so I guess I can close https://github.com/adrianriobo/qenvs/pull/143

File: .github/workflows/podman-desktop-e2e-windows.yaml (hunk 1 above)
Comment (odockal) [reply .github/workflows/podman-desktop-e2e-windows.yaml:91]: #19 should explain this. We need to install podman via podman desktop using GOAX in order to be able to run podman desktop e2e playwright tests (pulling images, starting containers, etc.).
I do not think we will remove this step at all.

File: .github/workflows/podman-desktop-e2e-windows.yaml (hunk 2 above)
Comment (odockal) [reply .github/workflows/podman-desktop-e2e-windows.yaml:117]: thx. fixed.

File: .github/workflows/podman-desktop-e2e-windows.yaml (hunk 3 above)
Comment (odockal) [reply .github/workflows/podman-desktop-e2e-windows.yaml:73]: > From the [script](https://github.com/odockal/pde2e-builder/blob/5dd843ac31d732341289e3a3ae8a000a89783720/lib/windows/run.ps1#L85C34-L85C41) I see you build PD based on fork and branch so you would be able to build from PR, but should not be easier to just pass the PR-Number?

Not intended for PR check now. 

> 
> Also I see you install dependencies without choco so I guess I can close [adrianriobo/qenvs#143](https://github.com/adrianriobo/qenvs/pull/143)

Yes please.

File: .github/workflows/podman-desktop-e2e-windows.yaml (hunk 1 above)
Comment (adrianriobo) [reply .github/workflows/podman-desktop-e2e-windows.yaml:91]: If that step is not gonna be removed ever we can use this step as the one for running those test as so have only one flow, the other option is adding the podman machine steps to your runner WDYT?

**Review by adrianriobo — APPROVED** [review]

**Pull request merged**